test/bettingTest.js

```diff
@@ -4,17 +4,29 @@ contract("lupi", function(accounts) {
     var instance, ownerAddress;
     var gasUseLog = new Array();
     var salt = 0xdb8780d713083a9addb6494cfc767d6ef4b1358315737e06bbb7fd84cc493d1c;
+    var requiredBetAmount = 1000000000000000000;
+    var ticketCountLimit = 2;
+    var feePt = 10000;
 
     before(function(done) {
-        lupi.deployed()
+        lupi.new(requiredBetAmount, ticketCountLimit, feePt)
```

Comment:
Haven't looked at the docs of this unit testing framework. Most provide a clear structure for structuring tests, eg. where to put repeated preparation code.